src/test/java/com/server/RequestHeaderParserTest.java

```diff
@@ -14,6 +14,7 @@ public void getVerbReturnsGetHTTPVerb() throws UnsupportedEncodingException {
 
         String headerString = "GET / HTTP/1.1";
         String directory = "/foo";
+        String bodyString = "";
```

Comment:
Awesome set of new tests!! 
I noticed that this file has "String bodyString = """ but it is not being used in these tests - is it possible to remove this then?